magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -58,17 +57,101 @@
 from magicctapipe.utils import calculate_disp, calculate_impact
 from traitlets.config import Config
 
-__all__ = ["mc_dl0_to_dl1"]
+__all__ = ["Calibrate_LST", "Calibrate_MAGIC","mc_dl0_to_dl1"]
```

Comment:
Why putting `Calibrate_LST`and `Calibrate_MAGIC` in `__all__`? they are not imported anywhere. Actually, they do not belong here. You should put them in a separate module (e.g. call it `calib`) of MCP, and import them here to use them in `mc_dl0_to_dl1`. This script needs to define only one function, and that is `mc_dl0_to_dl1`.